plugins/tag_search/sphinx/kTagFlowManager.php

```diff
@@ -73,7 +79,8 @@ public function objectCreated (BaseObject $object)
     			/* @var $object categoryEntry */
      			$privacyContexts = $object->getPrivacyContext() != "" ? self::trimObjectTags($object->getPrivacyContext()) : array();
```

Comment:
If we change this code then need to refactor that the fallback will be to set $privacyContexts to be kEntitlementUtils::DEFAULT_CONTEXT; , hence no need to check count